src/base/bittorrent/session.cpp

```diff
@@ -330,7 +330,7 @@ Session::Session(QObject *parent)
     , m_altGlobalUploadSpeedLimit(BITTORRENT_SESSION_KEY("AlternativeGlobalUPSpeedLimit"), 10, lowerLimited(0))
     , m_isAltGlobalSpeedLimitEnabled(BITTORRENT_SESSION_KEY("UseAlternativeGlobalSpeedLimit"), false)
     , m_isBandwidthSchedulerEnabled(BITTORRENT_SESSION_KEY("BandwidthSchedulerEnabled"), false)
-    , m_saveResumeDataInterval(BITTORRENT_SESSION_KEY("SaveResumeDataInterval"), 3)
+    , m_saveResumeDataInterval(BITTORRENT_SESSION_KEY("SaveResumeDataInterval"), 60)
```

Comment:
Why was this allowed? Aren't 60 **minutes** too long to save fastresume?
No justification was given in the PR text for this text.

Reply:
See #9152.

Reply:
And especially this: https://github.com/qbittorrent/qBittorrent/issues/9152#issuecomment-402158644